### DISCUSSION TITLE
Let's raise funds to support the Ukraine 🇺🇦 humanitarian crisis

### DISCUSSION THREAD

**Discussion by quetzalliwrites** [quetzalliwrites/website#2] [Announcements]
# Let's raise funds to support the Ukraine 🇺🇦 humanitarian crisis

## 📰 ATTENTION 🗞  

I'm looking for organizers and/or volunteers interested in teaming together to run an open-source collective for the Ukraine crisis. The Ukraine people have specifically requested **medical** and **military support**.  _(This is because men under 60yrs are blocked from leaving Ukraine, so many women are refusing to leave their men behind. It is more likely many will chose to stay and fight back.)_ Thus, raising funds for their military and hospitals makes a LOT OF SENSE!

We can start by selecting which organizations from the following Ukraine list we will raise money for:
https://ukrainewar.carrd.co/ 

From the above link, I would recommend we focus on the following:

- **Medical support:** 
   - [Razom for Ukraine](https://www.facebook.com/RazomForUkraine/fundraisers) 
   - [Medical aid and humanitarian relief for Ukraine](https://www.facebook.com/donate/337101825010055/)
   - [Katya's fundraiser for Sunflower of Peace](https://www.facebook.com/donate/507886070680475/)
   - [Revived Soldiers Ukraine](https://www.rsukraine.org/)
   - [Госпітальєри Госпитальеры (Hospitallers)](https://www.facebook.com/hospitallers/posts/2953630548255167)
- **Military support:** 
   - [Come Back Alive  ](https://savelife.in.ua/en/donate/)
   - [Army SOS](https://armysos.com.ua/en/)
   - [Phoneix Wings](http://wings-phoenix.org.ua/en/about-fund/)
   - [Vostok SOS](https://vostok-sos.org/en/make-a-donation/)

## Steps involved in starting fundraising for a humanitarian crisis

1. Create discussion to identify where donation will go (in our case, which orgs we select). 
2. Once donation target has been identified, then the actual fundraising campaign begins. This means creating your fundraising page/template, in your platform of choosing. We must decide which platform we want to use to collect the donations OR if we simply want to direct donations directly to one of the orgs above. 
3. Once the fundraising page is done, ensure the page details fundraiser goals and WHERE money is going to. All fundraisers need to have this info or people won't feel safe donating.
4. Start the actual sharing links and asking people to donate
5. Once donation goal is reached, we send the money from our platform to the org we selected.
6. Announce to folks that donation has been sent to the orgs selected and show proof of funds sent.

## Identifying fundraiser volunteers
I am happy to take the lead to get this rolling, but I will love and appreciate more volunteers + wisdom on making this a successful campaign to help Ukraine. 

If you'd like to get involved, please comment below in the discussion and tell me what you think 💭 + what you'd like to get involved in! ❤️

**Comment by mbifulco** [comment]
Here to help.

**Comment by jansche** [comment]
I want to be part of this. <3

**Comment by smoya** [comment]
Adding more sources:

MFA just released this website https://war.ukraine.ua. Verified live news, and organisms people can help to.

**Reply by quetzalliwrites** [reply]
this is perfect!

**Comment by quetzalliwrites** [comment]
I think the best thing to do is to individually donate to our org of choice because that way the money will reach them faster.

**Comment by quetzalliwrites** [comment]
I have just received this message from a Ukrainian woman that just this Monday safely made it to Krakow, Poland.

I am relieved for her safety and wanted to share her direct message to me on what websites she recommends we foreigners donate to...

> If you are ready to be valuable in this war against democracy you can transfer money to support the Ukrainian army. Any help is important now. Official donating for foreigners are here https://savelife.in.ua/en/donate/ or https://bank.gov.ua/en/news/all/natsionalniy-bank-vidkriv-spetsrahunok-dlya-zboru-koshtiv-na-potrebi-armiyi.
> If you want to do more and have more time and resources to help please check this website: https://how-to-help-ukraine-now.super.site/.If you’re already helping - sending all my love to you❤️ God damn we all appreciate it.


I am donating to the first one and I will be making more donations in the next weeks/months, as the situation continues. 🇺🇦 💙

**Comment by quetzalliwrites** [comment]
@kinlane, Here's the latest on the fundraiser! I have received confirmation from a Ukrainian woman ☝️  which orgs we should donate to.
